rcl/src/rcl/arguments.c

```diff
@@ -40,6 +40,10 @@ extern "C"
 {
 #endif
 
+#define RCL_ENABLE_FLAG_PREFIX "--enable-"
+#define RCL_DISABLE_FLAG_PREFIX "--disable-"
+#define RCL_LOG_LEVEL_FLAG_SEPARATOR ":="
```

Comment:
I'm not sure you need this define ...
We already have logic to parse that

Reply:
Use lexer